automatic/pdfcreator/tools/chocolateyinstall.ahk

```diff
@@ -0,0 +1,68 @@
+DetectHiddenWindows, on
+__Welcome:
+; Welcome window
+WinWait, Setup - PDFCreator, Welcome to the PDFCreator Setup Wizard
```

Comment:
What if it doesn't show for some reason (crach for example). AHK script will stay in the tray. You need timeout here.